apps/public-reference/pages/account/applications.tsx

```diff
@@ -76,23 +76,25 @@ export default () => {
             <title>{t("nav.myApplications")}</title>
           </Head>
           <MetaTags title={t("nav.myApplications")} description="" />
-          <div className="p-16" style={{ background: "#f6f6f6" }}>
-            <DashBlocks>
-              <DashBlock title={t("account.myApplications")} icon={<HeaderBadge />}>
-                <Fragment>
-                  {applications.map((application, i) => (
-                    <AppStatusItem
-                      key={application.id}
-                      status="inProgress"
-                      application={application}
-                      setDeletingApplication={setDeletingApplication}
-                    ></AppStatusItem>
-                  ))}
-                  {applications.length == 0 && noApplicationsSection}
-                </Fragment>
-              </DashBlock>
-            </DashBlocks>
-          </div>
+          <section className="" style={{ background: "#f6f6f6" }}>
```

Comment:
That's strange, I don't know how the hardcoded background snuck in here. @slowbot Do you mind relocating that to the stylesheet and switch to the Tailwind colors?